src/inverseRobot.js

```diff
@@ -24,7 +24,16 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  for (const key in robot) {
+    if (!robot[robot[key]]) {
```

Comment:
[Checklist](https://github.com/mate-academy/js_inverse_robot/blob/master/checklist.md) 2nd item, what if value contains falsy value? empty string, null, false, etc. Try to fix your check

Reply:
Fixed